src/blobtree/MaxNode.ts

```diff
@@ -40,14 +40,9 @@ export class MaxNode extends Node {
                 self.addChild(c);
             });
         }
-
-        // temp consts to speed up evaluation by avoiding allocations
```

Comment:
this comment has disappeared but you should keep it, move it above temps' declarations